security/pkg/server/ca/server.go

```diff
@@ -39,18 +40,23 @@ type CertificateAuthority interface {
 	// Sign generates a certificate for a workload or CA, from the given CSR and cert opts.
 	Sign(csrPEM []byte, opts ca.CertOpts) ([]byte, error)
 	// SignWithCertChain is similar to Sign but returns the leaf cert and the entire cert chain.
-	SignWithCertChain(csrPEM []byte, opts ca.CertOpts) ([]byte, error)
-	// GetCAKeyCertBundle returns the KeyCertBundle used by CA.
+	SignWithCertChain(csrPEM []byte, opts ca.CertOpts) ([]string, error)
+	// SignWithCertChain is similar to Sign but returns the leaf cert and the entire cert chain.
 	GetCAKeyCertBundle() *util.KeyCertBundle
+	// GetCACertificatesFromMeshConfig returns the CACertificates in mesh config.
+	GetCACertificatesFromMeshConfig() []*meshconfig.MeshConfig_CertificateData
```

Comment:
trying to understand why do these need to be exposed by the interface? this is functionality that is specific to the RA. I don't see this being used anywhere in the ca path

Reply:
removed from ca path